xml/System/Index.xml

```diff
@@ -357,8 +394,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Converts the value of the current Index object to its equivalent string representation.</summary>
+        <returns>The string representation of the Index.</returns>
```

Comment:
```suggestion
        <returns>The string representation of the <see cref="T:System.Index" />..</returns>
```

Reply:
I modified your suggestion to remove the duplicate dot.